src/common/Siderail.ts

```diff
@@ -0,0 +1,16 @@
+export interface SiderailItem {
+  title: string
+  url: string
+  children?: SiderailItem[]
```

Comment:
wait then how would you render the child pages? I think it's fine because in the component itself we don't recurse into the children so even if >2 levels of children are passed in as props, they don't get rendered

<img width="323" alt="Screenshot 2024-03-22 at 11 33 30 PM" src="https://github.com/isomerpages/isomer-components/assets/77665182/f7696572-8db4-4388-9d97-1824d4bdadc3">

Reply:
[49a8cb6](https://github.com/isomerpages/isomer-components/pull/43/commits/49a8cb66630df7dca4f91fb88c0676f5e86bec2b)

improved prop naming [49a8cb6](https://github.com/isomerpages/isomer-components/pull/43/commits/49a8cb66630df7dca4f91fb88c0676f5e86bec2b)

ignoring > 2 nesting for now as the component doesn't allow for it anyway